components/script/dom/dissimilaroriginwindow.rs

```diff
@@ -73,7 +73,7 @@ impl DissimilarOriginWindow {
             window_proxy: Dom::from_ref(window_proxy),
             location: Default::default(),
         });
-        unsafe { DissimilarOriginWindowBinding::Wrap::<crate::DomTypeHolder>(cx, win) }
+        DissimilarOriginWindowBinding::Wrap::<crate::DomTypeHolder>(cx, win)
```

Comment:
There are a number of other users, too. Look in window.rs, debuggerglobalscope.rs, dedicatedworkerglobalscope.rs, paintworkletglobalscope.rs, serviceworkerglobalscope.rs, and testworkletglobalscope.rs.

Reply:
Oh, they all call JSContext::from_ptr, which is also unsafe. All of those calls can be replaced with GlobalScope:: get_cx() instead, which is a safe method.